sming/sming/services/SpifFS/spiffs_sming.c

```diff
@@ -46,7 +46,7 @@ spiffs_config spiffs_get_storage_config()
 	if (cfg.phys_addr == 0)
 		return cfg;
 	cfg.phys_addr += 0x3000;
```

Comment:
I have the same idea but had a lot of trouble getting it to work (also due to the fs size and my 4M board) that I left it in for now. Now easy to get it out.